staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -119,8 +119,10 @@ func (o WardleServerOptions) Config() (*apiserver.Config, error) {
 		return nil, err
 	}
 
-	if err := o.Admission.ApplyTo(&serverConfig.Config, serverConfig.SharedInformerFactory, serverConfig.ClientConfig, apiserver.Scheme, admissionInitializer); err != nil {
```

Comment:
Does `ApplyTo` not do the right thing with pluginnames == 0?

Reply:
it tries to set a default: `pluginNames = a.enabledPluginNames()`

Reply:
> it tries to set a default: pluginNames = a.enabledPluginNames()

Hmm... that actually seems correct.  Someone doesn't specify, then we want the default.  If someone specifies `""`, then we want empty.  Seems like you'll want to set `""`.

Reply:
Have to check. I would bet we only check for `len(...) = 0`, we don't know the difference between not specified and `""`.